tests/integration/api_v3/rpc/blocks.test.js

```diff
@@ -0,0 +1,441 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import moment from 'moment';
+
+const util = require('util');
+
+const config = require('../../../config');
+
+const {
+	request,
+} = require('../../../helpers/socketIoRpcRequest');
+
+const {
+	invalidParamsSchema,
+	emptyResultEnvelopeSchema,
+	emptyResponseSchema,
+	jsonRpcEnvelopeSchema,
+	metaSchema,
+} = require('../../../schemas/rpcGenerics.schema');
+
+const {
+	blockSchema,
+} = require('../../../schemas/api_v3/block.schema');
+
+const wsRpcUrl = `${config.SERVICE_ENDPOINT}/rpc-v3`;
+const getBlocks = async params => request(wsRpcUrl, 'get.blocks', params);
+
+describe('Method get.blocks', () => {
+	let refBlock;
+	beforeAll(async () => {
+		[refBlock] = (await getBlocks({ limit: 1, offset: 5 })).result.data;
+	});
+
+	describe('is able to retireve block lists', () => {
```

Comment:
Please ensure that the changes requested/suggested for file `tests/integration/api_v3/http/blocks.test.js` are also reflected for the RPC-based tests.